submissions/nwokejeme-immaculeta/dom/signup.js

```diff
@@ -0,0 +1,66 @@
+let submitForm = document.getElementById('validateForm'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    otherName = document.getElementById('otherName'),
+    phoneNumber = document.getElementById('phoneNumber'),
+    email = document.getElementById('email');
+    password = document.getElementById('password');
+    confirmPassword = document.getElementById('confirmPassword');
+
+    let validate = (field, event, regEx, result, output) => {
+        if (field.value === "") {
+            field.nextElementSibling.innerHTML = result;
+            event.preventDefault();
+            return false;
+        } else
+        if (!field.value.match(regEx)) {
+            field.nextElementSibling.innerHTML = output;
```

Comment:
let your `else` and `if` be on the same line. e.g `} else if (!field.value.match(regEx)) {`